buidl/op.py

```diff
@@ -784,97 +845,47 @@ def op_checkmultisigverify(stack, tx_obj, input_index):
     return op_checkmultisig(stack, tx_obj, input_index) and op_verify(stack)
 
 
-def op_checklocktimeverify(stack, locktime, sequence):
-    if sequence == 0xFFFFFFFF:
+def op_checklocktimeverify(stack, tx_obj, input_index):
+    sequence = tx_obj.tx_ins[input_index].sequence
+    locktime = tx_obj.locktime
```

Comment:
Should this method be part of the Tx class?

Reply:
CLTV is an op, though.